

---

uint8 stait_img
